meta/service.go

```diff
@@ -0,0 +1,60 @@
+package meta
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/hashicorp/raft"
```

Comment:
I don't have a preference. Whatever is the thing people usually use. Godep?

Sent from my iPhone

> On May 6, 2015, at 10:29 AM, Ben Johnson <notifications@github.com> wrote:
> 
> In meta/service.go:
> 
> > @@ -0,0 +1,60 @@
> > +package meta
> > +
> > +import (
> > +	"io/ioutil"
> > +	"net/http"
> > +
> > +	"github.com/hashicorp/raft"
> Yes, what vendoring tool do you want to use?
> 
> —
> Reply to this email directly or view it on GitHub.
>